src/App.vue

```diff
@@ -111,10 +114,10 @@
               </template>
             </v-col>
 
-            <v-col cols="12" lg="3" style="position: relative" class="mt-2">
+            <v-col cols="12" lg="3">
               <aside>
-                <affix relative-element-selector=".col-lg-9" :offset="{ top: 86, bottom: 12 }">
-                  <sbc-fee-summary
+                <affix relative-element-selector=".col-lg-9" :offset="{ top: 100, bottom: -100 }">
```

Comment:
The offset is a bit "voodoo". Through trial and error, I found that these values work better for most screen sized (except very narrow/short). There's an [existing ticket](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/9809) to look into this more closely.